rebar.config

```diff
@@ -31,6 +32,10 @@
     deprecated_function_calls, deprecated_functions
 ]}.
 
+{overrides, [
```

Comment:
why do we need it?

Reply:
Because https://github.com/for-GET/jesse/blob/cf075d213ae9e9c54a748c93cc64d5350e646f9a/rebar.config#L11
and OTP26.1 throws warnings for matching on 0.0, as in the future +0.0 and -0.0 will not be considered equal anymore.